src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard-information.component.html

```diff
@@ -54,7 +54,7 @@ <h4 class="card-title">{{ 'artemisApp.assessmentDashboard.informationAboutYourAs
             </div>
         </div>
     </div>
-    <div class="col-md-4">
+    <div *ngIf="complaintsEnabled || feedbackRequestEnabled || isExamMode || true" class="col-md-4">
```

Comment:
This has `|| true`, so will always render?

Reply:
ah I forgot to remove it after debugging